.gitignore

```diff
@@ -76,3 +76,8 @@ WexLogFileOutput/*
 #==============================================================================#
 bindings/go/llvm/llvm_config.go
 bindings/go/llvm/workdir
+
+#==============================================================================#
+# Default output path use by Visual Studio cmake project builds.
+#==============================================================================#
+out/
```

Comment:
Fine, but seems like it should be an independent change.  Plus no newline at end (don't know if we care about that though).